### ISSUE TITLE
Sev-3 - Android - Talkback not reading a decision letter that is scanned, can read non-scanned document

### ISSUE THREAD

**Issue by rbontrager** [department-of-veterans-affairs/va-mobile-app#8186]
## What happened?

User +12 has quite a few decision letters that have been scanned in.  Talkback for android is not reading them.  It does read on iOS.  It would be nice to investigate if this could be read in on android as well.

## Specs:

- Device:
- OS:
- User Account:
- Accessibility State:

## Steps to Reproduce

1. Login to user +12
2. Navigate to claims > claim letters > select august 28,2020 letter. 
3. Turn on talkback.  Notice that the letter is not read out loud. 

## Bug Severity - BE SURE TO ADD THE SEVERITY LABEL
 
See [Bug Tracking](https://department-of-veterans-affairs.github.io/va-mobile-app/docs/QA#issue-severity) for details on severity levels

- Impact: High Low 
- Frequency: High Low 

- 3 - Low
- 2 - High
- 1 - Critical

## Screen shot(s) and additional information

<details>
  <summary>Full JSON response for services related to issue (expand/collapse)</summary>
  

</details>

## Ticket Checklist
- [x] Steps to reproduce are defined
- [ ] Desired behavior is added
- [x] Labels added (front-end, back-end, relevant feature, accessibility, etc)
- [x] Estimate of 1 added (for front-end tickets)
- [x] Added either to the relevant feature epic (if found during new feature implementation), or the [bug epic](https://app.zenhub.com/workspaces/va-mobile-product-view-610035bc5395bb000e62e529/issues/gh/department-of-veterans-affairs/va-mobile-app/5367)

**Comment by Sparowhawk** [comment]
@rbontrager for this one is this the pdf viewer? To further clarify is this after a user "downloads" the letter that there is an issue?

If that is the case then this is a library issue and it was last updated over 2 years ago. So I don't think we are going to get any additional support or fixes.

We could investigate looking at this library https://github.com/cyntler/react-doc-viewer which was last updated in November. It appears to have some additional benefits such as internationalization/multiple language support.

**Comment by rbontrager** [comment]
@Sparowhawk I think we discussed this privately but yes it is when the user "downloads" the letter. It only happens when its a scanned document, documents that aren't scanned work fine.

**Comment by Sparowhawk** [comment]
Ok. Then this ticket is going to be an investigation of that newer library and see if that potentially would fix our problems as the current one we use isn't supported.

**Comment by Sparowhawk** [comment]
@rbontrager Can you point me in the direction of one that isn't and one that is scanned so I can play around with the libraries?

**Comment by rbontrager** [comment]
The top letter of user +12 (august 28, 2020) is a scanned document. The very bottom letter (january 25, 2019) looks to be a normal not scanned in document.

**Comment by Sparowhawk** [comment]
@rbontrager I have tried react native pdf and react native pdf view as alternative libraries and they also do not work for Android. I am certain this is a common issue with the Android OS itself and in particular some comments say this was introduced with Android 11 which was released in Sep. 2020.

I was unable to find more then this. The only "alternative" that we can potentially do is we would need to have the upstream service and then the BE team add a field for a "label" that just has the contents as it is supposed to be read and we supply a wrapper view that would read the contents. Otherwise this is blocked externally by any of these libraries and the Android OS in general.

**Comment by timwright12** [comment]
> The only "alternative" that we can potentially do is we would need to have the upstream service and then the BE team add a field for a "label" that just has the contents as it is supposed to be read and we supply a wrapper view that would read the contents. Otherwise this is blocked externally by any of these libraries and the Android OS in general.

@Sparowhawk do we need API support for this to move it forward? cc @StacyB2023 @jperk51
